package.json

```diff
@@ -6,7 +6,8 @@
   "scripts": {
     "start": "node .",
     "stop": "pkill levweb",
-    "dev": "NODE_ENV=development nodemon .",
+    "dev": "npm run mock-kc-proxy & NODE_ENV=development nodemon .",
```

Comment:
When I do:
```
npm run dev
Ctrl+c
```
It kills both the app and the mock-kc-proxy. Does it not die on your machine?